dockstore-webservice/src/main/resources/migrations.1.9.0.xml

```diff
@@ -97,4 +97,10 @@
             <column name="os" type="varchar(255 BYTE)"/>
         </addColumn>
     </changeSet>
+
+    <changeSet author="esoth (generated)" id="1586206601202-8">
```

Comment:
was better with a descriptive id (used when debugging migrations in a live instance)